### PR TITLE
Add compat data for HTMLContentElement

### PR DESCRIPTION
**Pull request by bunnybooboo**
Not sure about the note on Fx as it ONLY appears relevant to Release channel. Added comment to enquire with Chris in spreadsheet.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLContentElement

### CODE REVIEW

**Comment by Elchi3** [comment]
@chrisdavidmills Can you have a look? I think this is shadow dom / web components related. Thanks!

File: api/HTMLContentElement.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            "version_added": "37"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "28"
```
Comment (ExE-Boss) [inline api/HTMLContentElement.json]: This should be:
```
{
  "version_added": "28",
  "version_removed": "59",
  "flags": [
    {
      "type": "preference",
      "name": "dom.webcomponents.enabled",
      "value_to_set": "true"
    }
  ]
}
```

File: api/HTMLContentElement.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+          },
+          "firefox_android": {
+            "version_added": "28"
```
Comment (ExE-Boss) [inline api/HTMLContentElement.json]: ditto

File: api/HTMLContentElement.json [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+              "version_added": "37"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "28"
```
Comment (ExE-Boss) [inline api/HTMLContentElement.json]: ditto

File: api/HTMLContentElement.json [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+            },
+            "firefox_android": {
+              "version_added": "28"
```
Comment (ExE-Boss) [inline api/HTMLContentElement.json]: ditto

File: api/HTMLContentElement.json [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
```
Comment (ExE-Boss) [inline api/HTMLContentElement.json]: ditto

File: api/HTMLContentElement.json [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+              "version_added": "37"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "28"
```
Comment (ExE-Boss) [inline api/HTMLContentElement.json]: ditto

**Pull request merged**